messaging/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>iaf-ibm</name>
+	<name>iaf-messaging</name>
```

Comment:
Je wou toch van dat `iaf` prefix af? Waarom dat niet gelijk doen voor deze module die je nu toch hernoemt?

Reply:
Wil ik straks in eens doen voor alle modules